cmake/AddInputFileTests.cmake

```diff
@@ -33,10 +35,23 @@ function(add_single_input_file_test INPUT_FILE EXECUTABLE CHECK_TYPE TIMEOUT)
       --check-options --input-file ${INPUT_FILE}
       )
   elseif("${CHECK_TYPE}" STREQUAL "execute")
+    # Write command into shell script to make it easier to chain multiple
+    # commands
+    file(
+      WRITE
+      ${PROJECT_BINARY_DIR}/tmp/${CTEST_NAME}.sh
+      "\
+#!/bin/sh\n\
+${CMAKE_BINARY_DIR}/bin/${EXECUTABLE} --input-file ${INPUT_FILE}\n\
+exit_code=$?\n\
+${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/tools/CleanOutput.py -v \
+--input-file ${INPUT_FILE} --output-dir ${CMAKE_BINARY_DIR}\n\
```

Comment:
I'm not sure the output dir is guaranteed to be `CMAKE_BINARY_DIR`. Can you check that it doesn't depend on where in the build directory you run ctest?

Reply:
done